lib/gemstash/cli/info.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "gemstash"
+require "puma/control_cli"
+
+module Gemstash
+  class CLI
+    # This implements the command line setup task:
```

Comment:
"This implements the command line info task"

Reply:
Sorry for that, I mis-read it as describing the setup command, fixed.